doc/rtd/topics/datasources/openstack.rst

```diff
@@ -83,3 +83,11 @@ including how it can be disabled by users on instances, see
 :doc:`/topics/vendordata`.
 
 .. vi: textwidth=78
+
```

Comment:
Nit: The vi comment (the line above this one) should stay at the bottom of the file.